entity-framework/core/modeling/entity-types.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Entity Types - EF Core
+author: roji
```

Comment:
'description:' would help with SEO, but it's optional

Reply:
Thanks! Will add.